### PR TITLE
Added Attributes for Classes

### PR DESCRIPTION
**Pull request by ShikharJ**
Ping @isuruf.

### CODE REVIEW

File: symengine/lib/symengine_wrapper.pyx [hunk 1]
```diff
@@ -766,6 +774,10 @@ cdef class Basic(object):
         return False
 
     @property
+    def is_number(self):
+        return False
```
Comment (isuruf) [inline symengine/lib/symengine_wrapper.pyx]: Return `None` here because you don't know whether it's a `number` or not.

**Comment by ShikharJ** [comment]
Ping @isuruf.

**Comment by ShikharJ** [comment]
Ping @isuruf.

File: symengine/lib/symengine_wrapper.pyx [hunk 2]
```diff
@@ -1458,6 +1556,14 @@ cdef class Complex(Number):
 
 class Infinity(Number):
 
+    @property
+    def is_infinite(self):
+        return True
+
+    @property
+    def is_prime(self):
```
Comment (isuruf) [inline symengine/lib/symengine_wrapper.pyx]: Why is this defined here and `zoo` and nowhere else?

Comment (ShikharJ) [inline symengine/lib/symengine_wrapper.pyx]: Most of the attributes directly follow from `SymPy`'s implementation. Though this is not needed currently, I thought it'd be helpful in having this as well (should an issue arise later on).

Comment (isuruf) [inline symengine/lib/symengine_wrapper.pyx]: I'm just wondering why having this here would be useful at all if `is_prime` is defined for classes with only `False`.

Comment (ShikharJ) [inline symengine/lib/symengine_wrapper.pyx]: I'll remove it then.

File: symengine/lib/symengine_wrapper.pyx [hunk 3]
```diff
@@ -778,10 +790,38 @@ cdef class Basic(object):
...
+    def is_finite(self):
+        return None
+
+    @property
     def is_Derivative(self):
         return False
 
     @property
+    def is_AlgebraicNumber(self):
+        return False
```
Comment (isuruf) [inline symengine/lib/symengine_wrapper.pyx]: If you are defining it here, where's the function that's supposed to return `True`?

Comment (ShikharJ) [inline symengine/lib/symengine_wrapper.pyx]: `AlgebraicNumber` is a class in itself in `SymPy`, with no child classes. Only a small fraction of code uses this flag, and the passed object is never an `AlgebraicNumber`. Hence, I didn't thought it'd be useful in implementing the class.

**Comment by ShikharJ** [comment]
Ping @isuruf.

**Pull request merged**